core/utils.py

```diff
@@ -15,7 +15,9 @@ def get_coordinates_for_city(city, country):
 
     try:
         data = req.json()[0]
-        return f'{data["lat"]}, {data["lon"]}'
+        formatted_lat = "{:.6f}".format(float(data["lat"]))
+        formatted_lon = "{:.6f}".format(float(data["lon"]))
+        return f'{formatted_lat}, {formatted_lon}'
```

Comment:
The ruff build is failing because of the single quotes here. May you please address this so we can merge this pull request. Sorry for taking so much time to review again.

Reply:
Hi @amakarudze ,
Thank you for your review. No problem let me fix this and I will update the code soon.

Reply:
You're welcome. Thanks for your work on this.